dist/src/main/resources/application-elasticsearch.yaml

```diff
@@ -121,6 +121,17 @@ zeebe:
             maxCacheSize: 10000
 
 camunda:
+
+  # Unified Configuration
+  # TODO: Remove legacy properties once issue https://github.com/camunda/camunda/issues/34902 is
+  #  done
+  data:
+    secondary-storage:
+      type: 'elasticsearch'
+      elasticsearch:
+        url: 'http://localhost:9200'
+  # -----
```

Comment:
:wrench: The legacy properties for url and type  in this file can also be removed